### PR TITLE
Adds PI install command as an example

### PR DESCRIPTION
**Pull request by lamagnifica**
@xcompass we don’t really have a robust reference or directory of Blocks and their GitHub locations at this time. Does this approach, adding an example for the PI Block, make sense to you?

@catong @pdesjardins sanity check please

### CODE REVIEW

File: en_us/install_operations/source/configuration/install_xblock.rst [hunk 1]
```diff
@@ -29,6 +29,12 @@ To install an XBlock, follow these steps.
      pip install git+https://github.com/oppia/xblock.git@9f6b95b7eb7dbabb96b77198a3202604f96adf65#egg=oppia-xblock==0.0.0
 
 
+   An example that installs the Peer Instruction XBlock follows.
+
+   .. code-block:: bash
+
+     pip install ubcpi-xblock==0.4.4
```
Comment (xcompass) [inline en_us/install_operations/source/configuration/install_xblock.rst:39]: Looks good to me. Though I would change the line 21 and 23 to the following to reflect both examples:

```
L21: Obtain the GitHub location and commit or PyPi package name and version for the XBlock.
L23: Run ``pip`` with a GitHub link or PyPi package name to the XBlock.
```

Comment (lamagnifica) [inline en_us/install_operations/source/configuration/install_xblock.rst:39]: good point! thanks!

File: en_us/install_operations/source/configuration/install_xblock.rst [hunk 2]
```diff
@@ -18,17 +18,25 @@ Open edX platform, both of the following tasks must be completed.
 
 To install an XBlock, follow these steps.
 
-#. Obtain the GitHub location and commit for the XBlock.
+#. Obtain the GitHub location and commit, or PyPi package name and version,
+   for the XBlock.
 
-#. Run ``pip`` with a GitHub link to the XBlock.
+#. Run ``pip`` with a GitHub link or PyPi package name to the XBlock.
```
Comment (catong) [inline en_us/install_operations/source/configuration/install_xblock.rst]: "to the XBlock" doesn't quite work after the addition of "or PyPi package name". I'd suggest "...with a GitHub link to or PyPi package name for the XBlock."

**Comment by catong** [comment]
@lamagnifica One suggestion, otherwise :+1:  from me.

**Pull request merged**